mypyc/primitives/generic_ops.py

```diff
@@ -241,6 +241,14 @@
     c_function_name='CPyObject_Size',
     error_kind=ERR_NEVER)
 
+# len(obj)
+# same as generic_len_op, however return py_ssize_t
+generic_ssize_t_len_op = custom_op(
+    arg_types=[object_rprimitive],
+    return_type=c_pyssize_t_rprimitive,
+    c_function_name='PyObject_Size',
+    error_kind=ERR_NEVER)
```

Comment:
This should be `ERR_NEG_INT`, since this returns -1 on error. Actually `CPyObject_Size` should also be `ERR_MAGIC`, looks like.

Reply:
Fixed!